WHATSNEW.rst

```diff
@@ -0,0 +1,18 @@
+=========================
+What's new in Crash Stats
+=========================
+
+Important changes to the Crash Stats service.
+
+
+No longer collecting Email data (2021-02-11)
+============================================
+
+*February 11th, 2021*
+
+As of February 11th, 2021, Socorro is no longer collecting Email data. Email
+data in crash reports collected on and after February 11th, 2021 will have the
+Email data stripped from the crash report before being saved and processed.
+
+One consequence of this is that new crash reports will no longer show up in
+Your Profile.
```

Comment:
This allows us to add service-related things to this file and they'll show up on the What's New page as well as in the docs on readthedocs.

This seems like a good-enough system. I'm not entirely sure what kinds of things should go in this file, but for now I'm saying it's "service changes". For example, dropping the Email field is a big service change. Other possible examples:

* adding/removing an API
* adding/removing a report
* switching to a new authentication system

We've done things like that in the past. It might be interesting to backfill the news.